native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/grpc-native/native-image.properties

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+Args =  -H:MaxDuplicationFactor=40.0
```

Comment:
Do we need this higher value to run the tests or a sample bbe?

Reply:
Only gave an error when running the module tests

Reply:
Anyway, since lang runtime sets this value 40, we do not require this config. Shall we remove this?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/1033/commits/944860cbf615140a09ccbec4513772f72435fa3a